Tangem/Common/UI/FeeRowView/FeeRowViewModel.swift

```diff
@@ -16,40 +16,51 @@ struct FeeRowViewModel: Identifiable {
     let option: FeeOption
     let isSelected: BindingValue<Bool>
 
-    var subtitleText: String? {
-        switch subtitle {
+    var cryptoAmount: String? {
+        switch formattedFeeComponents {
         case .loading:
             return ""
         case .loaded(let value):
-            return value
+            return value?.cryptoFee
         case .failedToLoad:
             return AppConstants.dashSign
         }
     }
 
+    var fiatAmount: String? {
+        switch formattedFeeComponents {
+        case .loading, .failedToLoad:
+            // Corresponding UI will be displayed by the cryptoAmount field
+            return nil
+        case .loaded(let value):
+            return value?.fiatFee
+        }
+    }
+
     var isLoading: Bool {
-        subtitle.isLoading
+        formattedFeeComponents.isLoading
     }
 
-    private let subtitle: LoadingValue<String?>
+    private let formattedFeeComponents: LoadingValue<FormattedFeeComponents?>
 
     init(
         option: FeeOption,
-        subtitle: LoadingValue<String?>,
+        formattedFeeComponents: LoadingValue<FormattedFeeComponents?>,
         isSelected: BindingValue<Bool>
     ) {
         self.option = option
-        self.subtitle = subtitle
+        self.formattedFeeComponents = formattedFeeComponents
         self.isSelected = isSelected
     }
 }
 
 extension FeeRowViewModel: Hashable {
     func hash(into hasher: inout Hasher) {
         hasher.combine(option)
-        hasher.combine(subtitle.isLoading)
-        hasher.combine(subtitle.value)
-        hasher.combine(subtitle.error != nil)
+        hasher.combine(formattedFeeComponents.isLoading)
+        hasher.combine(formattedFeeComponents.value??.cryptoFee)
+        hasher.combine(formattedFeeComponents.value??.fiatFee)
```

Comment:
`FormattedFeeComponents` надо сделать `Hashable` и здесь 
```suggestion
        hasher.combine(formattedFeeComponents.value)
```

без `??`